server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/rfc/model/ResponseObject.scala

```diff
@@ -0,0 +1,38 @@
+/** **************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ * *
+ * http://www.apache.org/licenses/LICENSE-2.0                 *
+ * *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ * ***************************************************************/
+
+package org.apache.james.jmap.rfc.model
+
+import play.api.libs.json.{JsArray, Json}
+
+object ResponseObject {
+
+  case class SessionState(value: String) extends AnyVal
+  case class Invocation(value: JsArray) extends AnyVal
+  case class ResponseObject(sessionState: SessionState, methodResponses: Seq[Invocation])
+
+  implicit val sessionStateFormat = Json.valueFormat[SessionState]
+  implicit val methodResponseFormat = Json.valueFormat[Invocation]
+  implicit val responseObjectFormat = Json.format[ResponseObject]
+
+  def deserialize(input: String): ResponseObject = {
```

Comment:
s/deserialize/serialize

Reply:
When saying `response` I think we would receive an object and serialize it into String. Here, you take a String and deserialize it into an object, what looks really alike `RequestObject.deserialize()`